data/dataProducts.js

```diff
@@ -0,0 +1,54 @@
+const products = [{
+		"id": "1",
+		"title": "Cinema",
+		"open":true,
```

Comment:
do not add `open:true` directly to the data. treat this data as if they came from backend you cannot modify.
you need to find a way to open the nodes after the data has been loaded into the widget